Resources/doc/reference/configuration.rst

```diff
@@ -57,6 +57,7 @@ Full Configuration Options
             title_logo:           bundles/sonataadmin/logo_title.png
             options:
                 html5_validate:       true
+                sort_admins:          false # Auto order groups and admins by label or id
```

Comment:
did you use ```app/console config:dump-reference```to generate the config? i would recommend that, i think it looks a little different when you use the ```info()```-method.